config/default.go

```diff
@@ -15,10 +15,6 @@ Host = "zkevm-bridge-db"
 Port = "5432"
 MaxConns = 20
 
-[ClaimTxManager]
```

Comment:
I think you should not remove the default configuration, as these values are still used in claimtxman.go and is safer to keep default values. What I would do is to add a field called "enabled" and use it as and indicator of whether you want to use it or not.